runtests.py

```diff
@@ -2145,6 +2145,8 @@ def main():
                       help="specify Pythran include directory. This will run the C++ tests using Pythran backend for Numpy")
     parser.add_option("--no-capture", dest="capture", default=True, action="store_false",
                       help="do not capture stdout, stderr in srctree tests. Makes pdb.set_trace interactive")
+    parser.add_option("--limited-api", dest="limited_api", default=False, action="store_false",
```

Comment:
This should say `"store_true"`, in which case the default already is `False`.

Reply:
Addressed